cli/src/commands/config/unset.rs

```diff
@@ -0,0 +1,36 @@
+use tracing::instrument;
+
+use super::ConfigLevelArgs;
+use crate::cli_util::get_new_config_file_path;
+use crate::cli_util::CommandHelper;
+use crate::command_error::user_error;
+use crate::command_error::CommandError;
+use crate::config::remove_config_value_from_file;
+use crate::config::ConfigNamePathBuf;
+
+/// Update config file to unset the given option.
```

Comment:
Should we document for users that unsetting keys, which are part of a table, might result in empty tables?

Reply:
I don't think explicit doc is needed.